Installation_Guide/Installation-Guide.rst

```diff
@@ -420,6 +422,65 @@ To uninstall, use the following command:
 Note: Ensure all other installed packages/components are removed.
 Note: Ensure all the content in the /opt/rocm directory is completely removed. If the command does not remove all the ROCm components/packages, ensure you remove them individually.
 
+
+.. _Arch Linux:
+
+
+Arch Linux
+^^^^^^^^^^^^
+
+The Arch Linux packages for ROCm are available on the `AUR`_ and are currently
+being maintained at `rocm-arch`_ by the Arch Linux community.
+
+Note: Support for ROCm on Arch Linux is currently experimental.
+That being said please submit any issues you encounter at the rocm-arch
+`issue tracker`_.
+
+**Installation**
+Since many packages will be installed, it is recommended to use an `AUR helper`_
+like `yay`_.
+
+It is also recommended to use the `arch4edu`_ binary repository as it will
+greatly speed up your installation time. `Add and use arch4edu as shown here`_.
```

Comment:
What about:

```
For directions see `Add arch4edu to your Archlinux`_.
```

Reply:
Ahh yeah that's better. Let me apply the change rq.